fast_barcode_scanner/example/lib/scanner_screen.dart

```diff
@@ -15,6 +15,20 @@ class ScannerScreen extends StatefulWidget {
 
 class _ScannerScreenState extends State<ScannerScreen> {
   final _torchIconState = ValueNotifier(false);
+  var _canChangeCamera = false;
+
+  @override
+  void initState() {
+    super.initState();
+    _checkCamera();
+  }
+
+  Future<void> _checkCamera() async {
```

Comment:
Naming sense here could be better, you're not checking the camera, you're checking if it's possible to change camera.

Reply:
suggest: _validateCameraSwitching